project-base/tests/FrontendApiBundle/Test/GraphQlWithLoginTestCase.php

```diff
@@ -11,28 +11,36 @@ abstract class GraphQlWithLoginTestCase extends GraphQlTestCase
     public const DEFAULT_USER_EMAIL = 'no-reply@shopsys.com';
     public const DEFAULT_USER_PASSWORD = 'user123';
 
+    /**
+     * @var string
+     */
+    protected $accessToken;
+
+    protected function setUp(): void
+    {
+        $this->client = $this->findClient(true);
+        $this->accessToken = $this->getAccessToken(static::DEFAULT_USER_EMAIL, static::DEFAULT_USER_PASSWORD);
+
+        parent::setUp();
```

Comment:
calling the parent setup at the end is intentional?

Reply:
Yes, the client at beginning is only created for accessToken and then can be rest of work handled to parentt